packages/schema/src/config/_app.ts

```diff
@@ -191,7 +195,8 @@ export default {
     meta: [],
     link: [],
     style: [],
-    script: []
+    script: [],
+    noscript: []
```

Comment:
Would you move this up to `app.head`? `meta` is deprecated and only here for backwards compatibility.
```suggestion
    script: []
```